server/server.go

```diff
@@ -61,10 +65,35 @@ func New(cfg *config.Config) (*Server, error) {
 		})
 	})
 
+	// Serve static content
+	workDir, _ := os.Getwd()
```

Comment:
This working directory should be defined by the cfg struct in https://github.com/kochman/hotshots/blob/45696548bda783f1b40baf08ea7a039cc6930fbc/config/config.go

You should follow the strategy for receiving configuration via env-var